wolfcrypt/test/test.c

```diff
@@ -25416,7 +25416,7 @@ static int ecc_encrypt_e2e_test(WC_RNG* rng, ecc_key* userA, ecc_key* userB,
 
     /* msg2 (response) from B to A */
     for (i = 0; i < (int)sizeof(msg2); i++)
-        msg2[i] = i + sizeof(msg2);
+        msg2[i] = (byte) (i + sizeof(msg2));
```

Comment:
Is this related to the cast issue in `ssl.c`?

Reply:
No, this is unrelated, this is just one of those cases where MSVS was complaining that without a typecast some data may be lost in the assignment. Unrelated to the other one in ssl.c